content/guides/saml.md

```diff
@@ -78,12 +78,11 @@ For more information about configuring specific IdP's, refer to the following Kn
 
 The following features can be enabled through the SAML Configuration dialog.
 
-### Just in Time Provisioning (JIT Provisioning)
-Normally users must be invited to Datadog, even for organizations with SAML enabled. 
-If a user that has not been invited to Datadog logs in via an Org's IdP, the SAML assertion will be validated, but they will be redirected to a SAML Error page.
+### Just-in-Time (JIT) Provisioning 
 
-Some organizations might not want to have to invite all of their users to Datadog. If you would like to make changes to how SAML works for your account, contact support.
-It is up to the organization to configure their IdP to not send assertions to Datadog if they don't want a particular user to access Datadog.
+With Just-in-Time provisioning, a user will be created within Datadog on the fly the first time they try to log in. This eliminates the need for admins to manually create user accounts one at a time.
+
+Some organizations might not want to invite all of their users to Datadog. If you would like to make changes to how SAML works for your account, please contact support. It is up to the organization to configure their IdP to not send assertions to Datadog if they don't want a particular user to access Datadog.
```

Comment:
can we link "contact" to the help page?

Reply:
@shashankpradhan ping?

Reply:
i think he meant https://docs.datadoghq.com/help/

Reply:
Done